pyodide_build/pypabuild.py

```diff
@@ -131,14 +128,17 @@ def remove_avoided_requirements(
 def install_reqs(
     build_env: Mapping[str, str], env: DefaultIsolatedEnv, reqs: set[str]
 ) -> None:
+    IGNORED_BUILD_REQUIREMENTS = get_host_build_flag(
+        "IGNORED_BUILD_REQUIREMENTS"
+    ).split()
```

Comment:
```suggestion
    IGNORED_BUILD_REQUIREMENTS = [pkg.strip() for pkg in get_host_build_flag(
        "IGNORED_BUILD_REQUIREMENTS"
    ).split()]
```

Let's strip the whitespace in each element.